tests/PHPStan/Analyser/AnalyserIntegrationTest.php

```diff
@@ -219,7 +219,7 @@ public function testBug2823(): void
 	public function testTwoSameClassesInSingleFile(): void
 	{
 		$errors = $this->runAnalyse(__DIR__ . '/data/two-same-classes.php');
-		$this->assertCount(4, $errors);
+		$this->assertCount(5, $errors);
```

Comment:
this one is interessting.. I am not sure this is correct, or whether a fix in the test-fixture is required: https://github.com/phpstan/phpstan-src/blob/e10a7aac373e8b6f21b430034fc693300c2bbb69/tests/PHPStan/Analyser/data/two-same-classes.php#L18

Reply:
Just update it two `rand(0, 1)`.

Reply:
doesnt work, since it turns into a fatal error then.

adjusted the expectations instead.

Reply:
👍